### ISSUE TITLE
Cannot Submit/detect quest objectives

### ISSUE THREAD

**Issue by Sukoshii** [Haggle1996/RevolutionPack#625]
Hello!
I'm sorry to bother you, I'm sure you're busy. But-- I've tried like crazy to find a situation similar to mine, and I've spoken to many people about it, but I just can't really seem to fix this problem and I was hoping maybe someone could help. 

For starters, this modpack is amazing and I absolutely love it and I love the questing system. BUT, it just so happens that my questing book has stopped working. This was a real bummer for me because I absolutely loved questing in this modpack. It was very rewarding and fun. 

To summarize, There are quest lines that you must do, and in order to progress, you have to finish the quest line before hand. WELL, I can't progress because, despite the fact that I've hand crafted the objects it's asking for, and that I have them in my inventory, it for some reason won't allow me to submit/detect them. Well, no, let me rephrase that. It let's me detect them, but it doesn't detect anything. 

I've done /bq_admin reset all [my username], which reset the quests, but then it bugged again on the same quests I've been unable to do. 

Something to note is that I couldn't get the Sluice to work, so I spawned in a Tiny Iron dust pile, and use it for the quest. Well, sure enough, I have a feeling that may have broken it. Granted, before that, I wasn't able to claim rewards for some quests, but at least I was able to finish the quest and progress. Perhaps me spawning in the iron dust pile could have broken it, but now I'd really like to fix it so I could possibly do that quest at a later time, legitimately.

If anyone has a way around the problem, or understands how the bq_admin stuff works, it would be greatly appreciated, because I really adore this modpack and I'd love to keep questing. I'll keep playing the modpack regardless of the questline, but it would still be nice to have a questline. =( 

Some of the quests that no longer work are: 
- Anything beyond Cooking for Blockheads
-The iron pickaxe one(despite me getting an iron pickaxe legitimately-- the only thing I cheated for was the dust pile.)
I'm sure there are a few more, I just haven't gotten the materials for them yet, or they're locked because I can't progress.

Thank you!

**Comment by Gamerthekid** [comment]
I am no mod packer but it sounds like their is something wrong with how
your launcher is reading the code. You possibly have another mod that is
interfering with the quest item, most likely you have an item with the same
id number as the quest item. I would suggest ( if your familiar with basic
coding) checking your logs and finding the red error bars and figuring
which one is the one about your item log error, or if you are lazy just
send it to the ATlauncher help website.

**Comment by Haggle1996** [comment]
Outside of realizing I had one missing thing from a different tab in the quest book (look for the arrows), this does sound completely weird. The quest book definitely works.
